charset/convert.go

```diff
@@ -3,23 +3,41 @@ package charset
 import (
 	"bytes"
 	"io"
+
+	"golang.org/x/text/encoding/charmap"
 )
 
-func ToUtf8(iso8859_1_buf []byte) string {
-	buf := make([]rune, len(iso8859_1_buf))
-	for i, b := range iso8859_1_buf {
-		buf[i] = rune(b)
+func mustConvert(b []byte, err error) []byte {
+	if err != nil {
+		panic(err)
 	}
-	return string(buf)
+	return b
+}
+
+// ToUtf8 decodes the provided buffer from the ISO8859_1 encoding to UTF8.
+// The result is returned as string.
+// Any error from the decoder will cause a panic.
+//
+// This function hasn't change its signature as it is used in almost any package.
+// To address the issues with panicing errors this package will most likely get
+// a new decoding function which return possible decoding errors.
+func ToUtf8(iso8859_1_buf []byte) string {
```

Comment:
Acronyms should be all upper case - see go naming conventions, e.g. https://talks.golang.org/2014/names.slide#6.

in this case, `ToUTF8`.